core/src/test/java/org/apache/iceberg/hadoop/TestHadoopTables.java

```diff
@@ -55,20 +54,19 @@ public class TestHadoopTables {
           required(1, "id", Types.IntegerType.get(), "unique ID"),
           required(2, "data", Types.StringType.get()));
 
-  @Rule public TemporaryFolder temp = new TemporaryFolder();
   private File tableDir = null;
```

Comment:
I think we can just add a `@TempDir` annotation to `tableDir` and remove `@TempDir Path temp` alltogether?

Reply:
I have switched to @TempDir tableDir